lib/app/service/docker.go

```diff
@@ -137,7 +137,12 @@ func (r *layerExporter) pushCmd(name, tag string) error {
 		Tag:  tag,
 	}
 	r.Infof("Pushing %v.", opts)
-	return r.dockerClient.PushImage(opts, dockerapi.AuthConfiguration{})
+	// Workaround a registry issue after updating go-dockerclient, set the password field to an invalid value so the
+	// auth headers are set.
+	// https://github.com/moby/moby/issues/10983
```

Comment:
It's something that's seen within go-dockerclient which is the first approach I tried at pushing these images, when talking to unauthenticated registries like the one we setup for image vendoring with some combination of vendoring updates. Even though the registry is unauthenticated, it will deny access if the auth header is missing. 

It's not really needed for this PR, but I figured I should leave it in because someone will inevitably hit this problem in the future.